docs/source/algorithms/FilterByLogValue-v1.rst

```diff
@@ -43,6 +43,25 @@ the range, then all events will be kept.
    :ref:`FilterByLogValue <algm-FilterByLogValue>` is not suitable for
    fast log filtering.
 
+
+For SNS Users
```

Comment:
This section is less for SNS users, and more for workspaces where the DAS logs on change, rather than periodically. Changing this heading to "Logs that only record changes" might be better. The text in the section makes more sense in the context of *how* ORNL DAS is special.